stdlib/public/core/DebuggerSupport.swift

```diff
@@ -328,6 +328,26 @@ public enum _DebuggerSupport {
 
     return target
   }
+
+  public static func stringForPrintObject(_ pointer: UnsafeRawPointer?, mangledTypeName: String) -> String {
+    guard let pointer else { return "invalid value pointer" }
+
+    guard let type =
+      unsafe _getTypeByMangledNameInContext(
+        mangledTypeName,
+        UInt(mangledTypeName.count),
+        genericContext: nil,
+        genericArguments: nil)
+      else {
+        return "invalid type \(mangledTypeName)"
+      }
+
+    func loadPointer<T>(type: T.Type) -> Any {
```

Comment:
I'm not sure what would happen here if T is noncopyable. Presumably it would be OK not to support noncopyable types as a first implementation, but we should make sure they at least do something vaguely sensible when failing. I don't think we can use reflection on them at all, but we could potentially check for noncopyable types and return the demangled type name and some message about not being able to show the contents.